projects/storefrontlib/src/layout/launch-dialog/config/launch-config.ts

```diff
@@ -1,8 +1,15 @@
 import { OutletPosition } from '../../../cms-structure/outlet/outlet.model';
+import { BREAKPOINT } from '../../config/layout-config';
 
 export abstract class LaunchConfig {
   launch?: {
-    [key: string]: LaunchOptions;
+    [key: string]: {
+      default: LaunchOptions;
```

Comment:
I guess in most cases we don't need the breakpoint specific configuration. Could we drop the default and only use breakpoint specific configurations optionally? Similarly to what we do in the pageslots?